### PR TITLE
wmco: remove ci-index-wmco-bundle creation

### PR DESCRIPTION
**Pull request by wgahnagl**
stop building the ci-index for the wmco bundle as it's deprecated

### CODE REVIEW

**Comment by aravindhp** [comment]
/pj-rehearse ci/rehearse/openshift/windows-machine-config-operator/master/ci-bundle-wmco-bundle

**Comment by aravindhp** [comment]
/pj-rehearse ci/rehearse/openshift/windows-machine-config-operator/master/images

**Comment by aravindhp** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-ci-bundle-wmco-bundle pull-ci-openshift-windows-machine-config-operator-master-images

**Comment by sebsoto** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-ci-bundle-wmco-bundle pull-ci-openshift-windows-machine-config-operator-master-images

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.15.yaml [hunk 1]
```diff
@@ -31,6 +31,7 @@ operator:
   - as: wmco-bundle
     context_dir: .
     dockerfile_path: bundle.Dockerfile
+    skip_building_index: true
```
Comment (aravindhp) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.15.yaml:34]: This is not needed

File: ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.16.yaml [hunk 2]
```diff
@@ -31,6 +31,7 @@ operator:
   - as: wmco-bundle
     context_dir: .
     dockerfile_path: bundle.Dockerfile
+    skip_building_index: true
```
Comment (aravindhp) [inline ci-operator/config/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-release-4.16.yaml:34]: This is not needed

**Review by aravindhp — CHANGES_REQUESTED** [review]

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-images

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-release-4.11-images pull-ci-openshift-windows-machine-config-operator-release-4.12-images pull-ci-openshift-windows-machine-config-operator-release-4.13-images pull-ci-openshift-windows-machine-config-operator-release-4.14-images

**Review by aravindhp — APPROVED** [review]
/lgtm

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-ci-bundle-wmco-bundle pull-ci-openshift-windows-machine-config-operator-release-4.11-ci-bundle-wmco-bundle pull-ci-openshift-windows-machine-config-operator-release-4.12-ci-bundle-wmco-bundle pull-ci-openshift-windows-machine-config-operator-release-4.13-ci-bundle-wmco-bundle pull-ci-openshift-windows-machine-config-operator-release-4.15-ci-bundle-wmco-bundle

**Comment by wgahnagl** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-release-4.14-ci-bundle-wmco-bundle

**Comment by aravindhp** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-aws-e2e-upgrade

**Comment by aravindhp** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-vsphere-e2e-upgrade

**Comment by aravindhp** [comment]
/pj-rehearse pull-ci-openshift-windows-machine-config-operator-master-vsphere-e2e-upgrade

**Comment by aravindhp** [comment]
/pj-rehearse ack

The vSphere upgrade job is failing in the expected manner in "run bundle". However I am seeing the issue this PR is attempting to fix with the `security` job now that it is marked as required. So I suspect this PR just transfers the issue to the `ci-bundle-wmco-bundle` job. But lets get this merged given we do need to skip building the index and then figure out how to move forward.

**Pull request merged**